automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -222,21 +223,10 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
         val getBxPython =
           """import bx.bitset
             |bx.bitset.sys.copyright""".stripMargin
-        val sparkJob =
```

Comment:
Moved to a newly created set below and renamed as `sparkJobToSucceed` for clarity.